Tasks/Stralenia/Task4/Task4.py

```diff
@@ -0,0 +1,35 @@
+      
+with open("text.txt", "r", encoding="utf8") as f:
+    
+    length = int(input('Enter the string size more than 35:\n'))
+    if length < 35:
+        print('Incorrect size')    
+           
+    with open("correct_text.txt", "w", encoding="utf8") as w:
+        
+        for line in f:
+            
+            while line:
```

Comment:
the **while** works totally correct here, but it can be easily replaced with **if** which will be more readable and clear, not a mistake though